app/src/main/res/layout/patientsummary_newbpsummary_content.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Format this line to two separate lines

Reply:
I am using the standard Android XML formatting. If I manually formatted it to 2 separate lines and reformat the file again it resets to original state.